CMakeLists.txt

```diff
@@ -32,6 +34,7 @@ target_include_directories(_torch_tvm PUBLIC
     ${CMAKE_CURRENT_SOURCE_DIR}
     ${CMAKE_CURRENT_SOURCE_DIR}/torch_tvm
     ${TVM_DIR}/include
+    ${TVM_DIR}/src
```

Comment:
curious, what include needs this?

Reply:
This ones: https://github.com/pytorch/tvm/pull/73/files#diff-2941f7ba675a837c849ce41ea7e78ce7R4-R5